### DISCUSSION TITLE
ALVR just gives nonexistent errors?

### DISCUSSION THREAD

**Discussion by EvokerKing** [alvr-org/ALVR#492] [Q&A]
I have been trying to use ALVR, but on the VR it says there is an error and I should check my PC for more info. However when I go to check my PC, there isn't an error. It says that everything is perfectly fine but it clearly isn't.

**Comment by zmerp** [comment]
What error did you get on the headset?

**Reply by EvokerKing** [reply]
It is cut off by a lot but here is what I can see: `Connection error: t alvr\common\src\sockets\control_socket.rs:30: io error`.

**Reply by zmerp** [reply]
In the next version the message will be "The server has disconnected". Regarding why you got that error, there was a fatal network error and the stream has terminated. This is maybe because of heavy packet loss for an extended period of time.

**Reply by EvokerKing** [reply]
Why? I have tried this many times and it got the same result at the same time as every other time I tried. I was connected by Oculus Link cable when setting it up and never took it out.
